packages/client/src/App.js

```diff
@@ -13,24 +18,38 @@ import { Inbox } from './components/InboxMessageAdmin/Inbox.component';
 import { AuthContextProvider } from './firebase/AuthContext';
 import { AboutToolbox } from './components/AboutToolbox/AboutToolbox.component';
 
+const Wrapper = ({ children }) => {
+  const location = useLocation();
+  useLayoutEffect(() => {
+    document.documentElement.scrollTo({
+      top: 100,
```

Comment:
Are you sure we want it to have a 100px offset from the top and left? 😅 
Shouldn't it be top: 0, left: 0?